src/Cardano/Shell/Constants/Types.hs

```diff
@@ -259,21 +274,25 @@ data Block = Block
 
 --- | Top-level Cardano SL node configuration
 data Node = Node
-    { noNetworkConnectionTimeout     :: !Int
-      -- ^ Network connection timeout in milliseconds.
-    , noConversationEstablishTimeout :: !Int
-      -- ^ Conversation acknowledgement timeout in milliseconds.
-    , noBlockRetrievalQueueSize      :: !Int
-      -- ^ Block retrieval queue capacity.
-    , noPendingTxResubmissionPeriod  :: !Int
-      -- ^ Minimal delay between pending transactions resubmission.
-    , noWalletProductionApi          :: !Bool
-      -- ^ Whether hazard wallet endpoint should be disabled.
-    , noWalletTxCreationDisabled     :: !Bool
-      -- ^ Disallow transaction creation or re-submission of
-      -- pending transactions by the wallet.
-    , noExplorerExtendedApi          :: !Bool
-      -- ^ Enable explorer extended API for fetching more.
+    { noSystemStartTime                 :: !Integer
+    -- ^ Node system start time.
+    , noSlotLength                      :: !Integer
+    -- ^ Slot length time.
+    , noNetworkConnectionTimeout        :: !Int
+    -- ^ Network connection timeout in milliseconds.
+    , noConversationEstablishTimeout    :: !Int
+    -- ^ Conversation acknowledgement timeout in milliseconds.
+    , noBlockRetrievalQueueSize         :: !Int
+    -- ^ Block retrieval queue capacity.
+    , noPendingTxResubmissionPeriod     :: !Int
+    -- ^ Minimal delay between pending transactions resubmission.
```

Comment:
Check with the network team about all these 4 params. The set of tunables will be different from the old node.

Reply:
* `noConversationEstablishTimeout` - I don't think we need it; we can relay on TCP connection timeouts. @dcoutts I don't this we need to timeout `connect`, this only complicates things, and it is configurable on the system level.
* `noConversationEstablishTimeout` - we'll need something like this for the handshake protocol, let's call it `noHandshakeTimeout`.  We'll use it for both sides of the handshake protocol: waiting for the initial message and waiting for the replay.
* `noBlockRetrievalQueueSize` - not needed
* `noPendingTxResubmissionPeriod` - not needed